cmd/test/main_test.go

```diff
@@ -276,6 +278,18 @@ func test_testCommand_only_k8s_files(t *testing.T, k8sValidator *K8sValidatorMoc
 	k8sValidator.AssertCalled(t, "GetK8sFiles", mock.Anything, 100)
 }
 
+
+func test_testCommand_demo_yaml(t *testing.T, evaluator *mockEvaluator, k8sValidator *K8sValidatorMock, filesConfigurations []*extractor.FileConfigurations, evaluationId int, ctx *TestCommandContext) {
+	test(ctx, []string{"8/*"}, TestCommandFlags{OnlyK8sFiles: true, Output: ""})
+
+	k8sValidator.AssertCalled(t, "ValidateResources", mock.Anything, 100)
+	k8sValidator.AssertCalled(t, "GetK8sFiles", mock.Anything, 100)
+	evaluator.AssertCalled(t, "CreateEvaluation", "134kh", "", "1.18.0", "Default")
+	evaluator.AssertCalled(t, "Evaluate", filesConfigurations, mock.Anything, mock.Anything)
```

Comment:
Hi Kanika! Look like its your first PR, woot woot 🥳🥳

I'm not sure that using `mock.Anything` is right. Since almost everything is mocked we can control the returned values and check that we pass them along. For instance, we can test that the actual response from `CreateEvaluation` is passed to `Evaluate`. More importantly, this test should check that the demo flow works. It's has huge impact for us :) To test this we can check that the invocation results object is as expected. Since it's the demo flow, we can know for sure what will be printed(the file is located at `fixtures/kube/k8s-demo.yaml`).

Reply:
Hey, @noaabarki  So as per the discussion I have removed the "mock.anything"  term everywhere from the test...Now should I push the changes?

Reply:
Certainly😊

Reply:
I have pushed the changes...

Reply:
Hey @Kanika637, everything looks good. May you please review and fix the linting issues?

Reply:
Sure...Let me look into this

Reply:
Linter issues solved...Please review it

Reply:
> @Kanika637 following our conversation, would you please review the lint issues? cc @dimabru

Actually, I do make some commits and solve the linting issues and it shows me solved but after some time again it shows me the same error.. "cannot use (TestCommandFlags literal) (value of type TestCommandFlags) as *TestCommandOptions value in argument to test (type check)"
How to solve this?

Reply:
Hey @Kanika637, sorry for the late response. Seems like the problem is that you try to pass the command flags and initiate the struct in literal form. Try storing it in a variable instead

Reply:
Okay @noaabarki